.github/workflows/pr.yml

```diff
@@ -14,7 +14,7 @@ on:
 env:
   #we use this variable in ciRunSbt.sh
   #NOTE: for publishing we use different settings, we don't use ciRunSbt.sh there
-  CROSS_BUILD: ${{ github.ref == 'refs/heads/staging' || github.ref == 'refs/heads/master' }}
+  CROSS_BUILD: ${{ github.ref == 'refs/heads/staging' || github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/preview') }}
```

Comment:
Do we want to run all tests on preview branches? Publish is done for all versions regardless of this setting

Reply:
In fact we do some snapshot release, so we should test it.